scripts/scr_apothecary_ship/scr_apothecary_ship.gml

```diff
@@ -1,4 +1,4 @@
-function scr_apothecary_ship() {
+function scr_apothecary_ship(capital_number, frigate_number, escort_number) {
```

Comment:
Like scr_add_corruption we have undefined capital_number etc here we can declare them in obj_all_fleet and inherit in obj_p_fleet which is much cleaner as it will remain relevant to this object going forward.

Declaring in obj_controller also works as this is where the script is called from.